tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{38,39,310}-dj32-wagtail52
     py{38,39,310,311}-dj42-wagtail{52,60}
-    py{311,312}-dj50-wagtail{52,60}
+    py{311,312}-dj{50}-wagtail{52,60}
+    py{311,312}-dj{51}-wagtail{60}
```

Comment:
given 6.0 is no longer supported, I would say replace with 6.1 and 6.2

Reply:
@zerolab Thanks, I will do that. But I think it will be best if #48 would be merged first and then I will rebase above it.

Reply:
Fair call. I went ahead and merged it as it was approved. Can you rebase?

Reply:
@zerolab Done. Seems like all tests has passed.